docs(element-templates): storage location of synced templates (#2109)
camunda/camunda-modeler: docs/element-templates/README.md

Element templates are defined as [JSON files](#defining-templates). There are two ways to integrate them into the Camunda Modeler:

> 1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Locally the sycned templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](https://github.com/camunda/camunda-modeler/tree/master/docs/search-paths#user-data-directory). Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well).

Reviewer: > sycned

Typo.

Author: ty :+1:

Reviewer: BTW I'm not sure if this is detailed enough to actually remove the templates manually.

> 1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well). Locally the synced templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](https://github.com/camunda/camunda-modeler/tree/master/docs/search-paths#user-data-directory). Making manual changes to the `config.json` is not recommended and should be a "power-user" action only.

Reviewer 2: ```suggestion
1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well). Locally the synced templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](https://github.com/camunda/camunda-modeler/tree/master/docs/search-paths#user-data-directory). You should not manually change the `config.json` unless you know what you're doing.
```

Reviewer 2: I'd not use the term _power-user_ as I'd argue that manually changing internals falls outside of the user category.
